src/odemis/gui/cont/tabs.py

```diff
@@ -1755,6 +1756,284 @@ def get_display_priority(cls, main_data):
+        streams = data_to_static_streams(data)
+        bbox = (None, None, None, None)  # ltrb in m
+        for s in streams:
+
+            # add stream to correlation controller
+            self._correlation_controller._add_correlation_stream(s)          
+            
+            # add stream to streambar
+            sc = self._streambar_controller.addStream(s, add_to_view=True, play=False)
+            sc.stream_panel.show_remove_btn(True)
+
+            # Compute the total bounding box
+            try:
+                s_bbox = s.getBoundingBox()
+            except ValueError:
+                continue  # Stream has no data (yet)
+            if bbox[0] is None:
+                bbox = s_bbox
+            else:
+                bbox = (min(bbox[0], s_bbox[0]), min(bbox[1], s_bbox[1]),
+                        max(bbox[2], s_bbox[2]), max(bbox[3], s_bbox[3]))
+                        
+        # Recenter to the new content only
+        if bbox[0] is not None: 
+            if isinstance(s, StaticSEMStream):
+                self.panel.vp_secom_tr.canvas.fit_to_bbox(bbox)
+            if isinstance(s, StaticFluoStream):
+                self.panel.vp_secom_tl.canvas.fit_to_bbox(bbox)
+
+
+    def _on_acquisition(self, is_acquiring):
+        # When acquiring, the tab is automatically disabled and should be left as-is
+        # In particular, that's the state when moving between positions in the
+        # Chamber tab, and the tab should wait for the move to be complete before
+        # actually be enabled.
+        if is_acquiring:
+            self._stage.position.unsubscribe(self._on_stage_pos)
+        else:
+            self._stage.position.subscribe(self._on_stage_pos, init=True)
+
+    # role -> tooltip message
+    DISABLED_TAB_TOOLTIP = {
+        "meteor": "Correlation can only be performed in FM mode",
+    }
+
+    def _on_stage_pos(self, pos):
+        """
+        Called when the stage is moved, enable the tab if position is imaging mode, disable otherwise
+        :param pos: (dict str->float or None) updated position of the stage
+        """
+        guiutil.enable_tab_on_stage_position(
+            self.button,
+            self.main_data.posture_manager,
+            self._allowed_targets,
+            tooltip=self.DISABLED_TAB_TOOLTIP.get(self.main_data.role)
+        )
+
+    def terminate(self):
+        super(CorrelationTab, self).terminate()
+        self._stage.position.unsubscribe(self._on_stage_pos)
+        # make sure the streams are stopped
+        for s in self.tab_data_model.streams.value:
+            s.is_active.value = False
+
+    @classmethod
+    def get_display_priority(cls, main_data):
+        if main_data.role in ("enzel", "meteor", "mimas"):
+            return 2
+        else:
+            return None
+
+    def Show(self, show=True):
+        assert (show != self.IsShown())  # we assume it's only called when changed
+        super(CorrelationTab, self).Show(show)
+
```

Comment:
If it's shown, and there is no stream yet, should it automatically load the FM overview streams from the localization tab? That would avoid the user having to locate them again in the file system.

Reply:
Would we want to always add the overview streams here? not just when nothing is shown

Reply:
As discussed, let's make it the same behaviour as the chamber tab: automatically add all the acquired overview.

We probably could automatically empty the streams in the correlation tab when the user changes project (as for the localization tab).